cypress/integration/artist.spec.js

```diff
@@ -20,6 +20,5 @@ describe("/artist/:id", () => {
     cy.get("h1").should("contain", "Pablo Picasso")
     cy.get("h2").should("contain", "Spanish, 1881–1973")
     cy.get("h2").should("contain", "Notable Works")
-    cy.get("h2").should("contain", "Iconic Collections")
```

Comment:
This is a lazy loaded rail outside of the initial view, triggered on scroll: 
<img width="848" alt="Screen Shot 2021-03-28 at 1 33 01 PM" src="https://user-images.githubusercontent.com/236943/112767221-50ff3380-8fca-11eb-8e74-02e3d108e24d.png">

Def not critical as we're only making sure the page loads